genestack_client/genestack_connection.py

```diff
@@ -254,7 +254,7 @@ def __invoke(self, path, post_data, trace=None):
         if trace:
             headers['Genestack-Trace'] = 'true'
         f = self.connection.open(path, post_data, headers=headers)
-        response = Response(json.load(f))
+        response = Response(json.loads(f.read(decode_content=True)))
```

Comment:
I have found that using `raw` is apparently not really document/recommended: https://github.com/requests/requests/issues/2155#issuecomment-50893660

Is that a problem for us?

Reply:
I have fixed this.  

`raw` is still used but only in deprecated method.